cdap-client/src/main/java/co/cask/cdap/client/ServiceClient.java

```diff
@@ -103,16 +103,15 @@ public ServiceSpecification get(Id.Service service)
   }
 
   /**
-   * Checks whether the {@link Service} is active.
+   * Checks whether the {@link Service} is active. Returns without throwing any exception if it is active.
    *
    * @param service ID of the service
-   * @return 'Active' message when service is active
    * @throws IOException if a network error occurred
    * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
    * @throws NotFoundException if the app or service could not be found
-   * @throws ServiceUnavailableException if the service has started but is not available right now
+   * @throws ServiceUnavailableException if the service is not available
    */
-  public String getAvailability(Id.Service service) throws IOException, UnauthenticatedException, NotFoundException,
+  public void checkAvailability(Id.Service service) throws IOException, UnauthenticatedException, NotFoundException,
```

Comment:
Seems like a lot of the cdap-clients are annotated `@Beta`.
This particular method - the return type was incorrect, according to the javadoc, and this method was only just recently introduced.
I think its fine to modify the return type to remove that incorrectness for this method.

In general, I think we should care about compatability in cdap-clients, since users may write external programs to manage their CDAP apps/programs with the,